tests/basics/generator_pend_throw.py.exp

```diff
@@ -2,4 +2,13 @@
 1
 raised ValueError()
 ret was: None
-TypeError
+raised OSError()
+raised ValueError('generator already executing',)
+raised ValueError('generator already executing',)
+0
+ignore CancelledError
+1
+raised CancelledError()
+None
+CancelledError()
+raised ValueError('exception',)
```

Comment:
now needs to be replaced with `raised ValueError()`

Reply:
Done